packages/api/tests/lambdas/ems-metadata-report.js

```diff
@@ -74,7 +52,33 @@ test.serial('generateReport creates flat file for collections in both CUMULUS an
   sinon.stub(CMRSearchConceptQueue.prototype, 'shift').callsFake(() => cmrCollections.shift());
 
   await new models.Collection().create(matchingColls.concat(extraDbColls));
+}
+
+test.beforeEach(async () => {
+  process.env.system_bucket = randomString();
+  process.env.stackName = 'test-stack';
+  process.env.ems_provider = 'testEmsProvider';
+  process.env.CollectionsTable = randomString();
 
+  await aws.s3().createBucket({ Bucket: process.env.system_bucket }).promise();
+  await new models.Collection().createTable();
```

Comment:
if you pass `t` as a parameter into the `async ()` call, then you can do something like `t.collectionModel = new models.Collection();`

Which you can then use as the collection model for both creating and deleting tables. I don’t think this is a necessary change, just another option.